websocket-resources/src/test/java/org/whispersystems/websocket/logging/WebSocketRequestLogTest.java

```diff
@@ -22,12 +23,18 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Locale;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 
 public class WebSocketRequestLogTest {
 
+  @Before
+  public void beforeEachTest() {
+    Locale.setDefault(Locale.US);
```

Comment:
```suggestion
    Locale.setDefault(Locale.ENGLISH);
```
The log output is the same, but our servers run with `locale=en`, so this will be less confusing.

Also, to minimize side effects (and avoid masking similar issues), we should store the value of the original default locale before overriding it and reset it in an `@After` method.